src/renderer/src/helpers/StringHelper.ts

```diff
@@ -25,4 +25,17 @@ export class StringHelper {
   static normalizeText(text: string) {
     return text.trim().toLowerCase()
   }
+
+  static validateName(name: string, maxLength: number = 30) {
```

Comment:
1. Use `validateValue` instead of `validateName`.
2. Use `value` instead of `name`.